README.md

```diff
@@ -401,6 +402,31 @@ apk del grep sed curl fzf git aria2 ffmpeg ncurses
 - ffmpeg - m3u8 Downloader (fallback)
 - fzf - User interface
 
+### Ani-Skip (optional)
+
+Ani-skip is a script to automatically skip anime opening sequences, making it easier to watch your favorite shows without having to manually skip the intros each time (from the original [README](https://github.com/synacktraa/ani-skip/tree/master#a-script-to-automatically-skip-anime-opening-sequences-making-it-easier-to-watch-your-favorite-shows-without-having-to-manually-skip-the-intros-each-time)).
+
+Ani-skip uses the external lua script function of mpv and as such – for now – only works with mpv.
+
+**Warning:** For now, ani-skip does **not** seem to work under Windows.
+
+**Note:** It may be, that ani-skip won't know the anime you're trying to watch (It uses the [aniskip API](https://github.com/lexesjan/typescript-aniskip-extension/tree/main/src/api/aniskip-http-client) and you can contribute missing anime or ask for including it in the database on their [discord server](https://discord.com/invite/UqT55CbrbE)).
+
+#### Usage
+
+By using the `--skip` flag, ani-cli runs the external ani-skip script if it is installed (if the dependency check fails, ani-cli exits).
+
+By setting the environment variable `ANI_CLI_SKIP_INTRO=1`, the feature can be enabled permanently.
```

Comment:
we haven't documented usage in readme for a long time, and since you have it in the manpage, this can be taken out too

Reply:
I removed the usage section as well.
Someone requested it to be in the readme if I remember right. ;)